.gitignore

```diff
@@ -78,6 +78,7 @@ ledger-api/.bin
 **/.history/
 ### Bazel: https://www.gitignore.io/api/bazel ###
 /bazel-*
+/compatibility/bazel-*
```

Comment:
I assume this was meant to ignore the `bazel-bin`-style symlinks, but as you do have a few legitimate files and folders starting with `bazel-` in this branch, it looks like this prevents Azure Pipelines from properly cleaning them when switching branches.

Reply:
The gitignore is fine. But bazel picks up the `bazel-bin` directory here as a regular file since it’s a different workspace. Fixed by #5687